env/wflow_gaea.env

```diff
@@ -0,0 +1,8 @@
+#Setup instructions for NOAA RDHPC Hera using Intel-18.0.5.274 (bash shell)
+
+module use /lustre/f2/pdata/esrl/gsd/contrib/modulefiles
+module load rocoto
+module load miniconda3
+export TEST_EXTRN_MDL_SOURCE_BASEDIR=/lustre/f2/dev/Mark.Potts/EPIC/SRW/model_data
```

Comment:
@mark-a-potts I think at least the `TEST_EXTRN_...` variable should be defined in a machine file at `ush/machine/gaea.sh`.  So that machine file needs to be created.

Reply:
You are correct. I did create that file and added the environment variable there, but forgot to pull it out of the wflow environment. I'll fix that.